### ISSUE TITLE
maskedTextFieldDelegate.put not working on 3.5.0

### ISSUE THREAD

**Issue by Robuske** [RedMadRobot/input-mask-ios#42]
The title says it all, `put(text: into:)` doesn't seem to be working on the new version.
I didn't have the time to do extensive testing but was happening on the iPhone SE simulator with iOS 10.
I saw that you changed how that part works because of the UITextView support, so maybe you forgot to call something?

(yes, I did a downgrade and 3.4.1 is working fine)

**Comment by taflanidi** [comment]
Hi @Robuske, thanks a lot for your report.
Now that's a bummer. Let me investigate this a little bit, and I'll comeback with a patch shortly.

I knew this "refactoring" will have consequences.

**Comment by taflanidi** [comment]
Okay, it looks like the problem is the old version of iOS.
I've launched our `Sample` project on iPhone SE, `11.4`. The code is altered as follows:

```swift
open override func viewDidLoad() {
    super.viewDidLoad()
    listener.affineFormats = [
        "8 ([000]) [000] [00] [00]"
    ]
    
    listener.put(text: "333", into: field) // this line is added
}
```
On app launch, everything works as expected:
<img width="432" alt="screen shot 2018-07-25 at 17 56 14" src="https://user-images.githubusercontent.com/6849363/43208996-523e0430-9034-11e8-8515-bf694636ed08.png">
While on `10.3.1` literally nothing happens:
<img width="432" alt="screen shot 2018-07-25 at 17 58 02" src="https://user-images.githubusercontent.com/6849363/43209025-5f726b5a-9034-11e8-9afa-4c074605b33b.png">

**Comment by Robuske** [comment]
Wow, looks like I was lucky to be testing with iOS 10 when I made the update. At least you found the problem =)

**Comment by taflanidi** [comment]
Fixed in `4.0.0`.
